packages/aws-client/src/StepFunctions.ts

```diff
@@ -2,18 +2,32 @@
  * @module StepFunctions
  */
 
+import {
+  DescribeExecutionInput,
+  DescribeExecutionOutput,
+  DescribeStateMachineInput,
+  DescribeStateMachineOutput,
+  GetExecutionHistoryInput,
+  GetExecutionHistoryOutput,
+  HistoryEvent,
+  ListExecutionsCommandInput,
+} from '@aws-sdk/client-sfn';
+
 import { sfn } from './services';
-import { improveStackTrace, retryOnThrottlingException } from './utils';
+
+import { retryOnThrottlingException } from './utils';
 import { inTestMode } from './test-utils';
 
+export { HistoryEvent, DescribeExecutionOutput } from '@aws-sdk/client-sfn';
```

Comment:
Looks like a duplicate import

Reply:
Lines 7 and 12 are imports, but Line 21 is an export. I believe Charles originally added this line, so that they type can be used by another package (orca-recovery-adapter) that depends on aws-client.

Reply:
Oh geez. Sorry, can't read.